### PR TITLE
Fix incorrect difficulty name and formatting in `4CWC2022`

### PR DESCRIPTION
**Pull request by Kimitakari**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/4CWC/2022/en.md [hunk 1]
```diff
@@ -113,19 +113,19 @@ The 4 Digit Catch World Cup 2022 is run by various community members.
...
   1. [BLANKFIELD - Retrospective City (Nelly) \[Collab Platter\]](https://osu.ppy.sh/beatmapsets/1203586#fruits/2531997)
   2. [petit milady - azurite (garden) \[ArThasCD's hard\]](https://osu.ppy.sh/beatmapsets/777993#fruits/1653983)
-- Mixed Mod
-  1. [trapman featuring CreeperWorks - minefield surfing (Nelly) \[Platter\]](https://osu.ppy.sh/beatmapsets/1819363#fruits/3733108)
+- MixedMod
+  1. [trapman featuring CreeperWorks - minefield surfing (Nelly) \[Salad\]](https://osu.ppy.sh/beatmapsets/1819363#fruits/3733108)
   2. [Nekomata Master - Byakuya Gentou (ignorethis) \[Hyper\])](https://osu.ppy.sh/beatmapsets/36849#fruits/119376)
 - Tiebreaker
-  1. [Camellia feat. Nanahira - Bassline Yatteru? w (Kukkai) \[Rain\])](https://osu.ppy.sh/beatmapsets/1643938#fruits/3356878)
+  1. **[Camellia feat. Nanahira - Bassline Yatteru? w (Kukkai) \[Rain\])](https://osu.ppy.sh/beatmapsets/1643938#fruits/3356878)**
```
Comment (ZeroPyrozen) [inline wiki/Tournaments/4CWC/2022/en.md:128]:
```suggestion
  1. **[Camellia feat. Nanahira - Bassline Yatteru? w (Kukkai) \[Rain\]](https://osu.ppy.sh/beatmapsets/1643938#fruits/3356878)**
```

File: wiki/Tournaments/4CWC/2022/en.md [hunk 2]
```diff
@@ -113,19 +113,19 @@ The 4 Digit Catch World Cup 2022 is run by various community members.
...
   2. [Yuna (CV: Kanda Sayaka) - Break Beat Bark! (Cut Ver.) (-Crimson-) \[Hard\]](https://osu.ppy.sh/beatmapsets/1285419#fruits/2670654)
-- Double Time
+- DoubleTime
   1. [BLANKFIELD - Retrospective City (Nelly) \[Collab Platter\]](https://osu.ppy.sh/beatmapsets/1203586#fruits/2531997)
   2. [petit milady - azurite (garden) \[ArThasCD's hard\]](https://osu.ppy.sh/beatmapsets/777993#fruits/1653983)
-- Mixed Mod
-  1. [trapman featuring CreeperWorks - minefield surfing (Nelly) \[Platter\]](https://osu.ppy.sh/beatmapsets/1819363#fruits/3733108)
+- MixedMod
+  1. [trapman featuring CreeperWorks - minefield surfing (Nelly) \[Salad\]](https://osu.ppy.sh/beatmapsets/1819363#fruits/3733108)
   2. [Nekomata Master - Byakuya Gentou (ignorethis) \[Hyper\])](https://osu.ppy.sh/beatmapsets/36849#fruits/119376)
```
Comment (ZeroPyrozen) [inline wiki/Tournaments/4CWC/2022/en.md:126]:
```suggestion
  2. [Nekomata Master - Byakuya Gentou (ignorethis) \[Hyper\]](https://osu.ppy.sh/beatmapsets/36849#fruits/119376)
```

File: wiki/Tournaments/4CWC/2022/en.md [hunk 3]
```diff
@@ -113,19 +113,19 @@ The 4 Digit Catch World Cup 2022 is run by various community members.
...
   1. [Yuzuki - Dear You (DJ Genericname DnB Remix) (R-34) \[R-34 & M0RU's Salad (4CWC Edit.)\]](https://osu.ppy.sh/beatmapsets/1819634#fruits/3733788)
   2. [Yuna (CV: Kanda Sayaka) - Break Beat Bark! (Cut Ver.) (-Crimson-) \[Hard\]](https://osu.ppy.sh/beatmapsets/1285419#fruits/2670654)
-- Double Time
+- DoubleTime
   1. [BLANKFIELD - Retrospective City (Nelly) \[Collab Platter\]](https://osu.ppy.sh/beatmapsets/1203586#fruits/2531997)
   2. [petit milady - azurite (garden) \[ArThasCD's hard\]](https://osu.ppy.sh/beatmapsets/777993#fruits/1653983)
-- Mixed Mod
-  1. [trapman featuring CreeperWorks - minefield surfing (Nelly) \[Platter\]](https://osu.ppy.sh/beatmapsets/1819363#fruits/3733108)
+- MixedMod
+  1. [trapman featuring CreeperWorks - minefield surfing (Nelly) \[Salad\]](https://osu.ppy.sh/beatmapsets/1819363#fruits/3733108)
```
Comment (ZeroPyrozen) [inline wiki/Tournaments/4CWC/2022/en.md:125]: Is this difficulty name change already changed in beatmap page?

File: wiki/Tournaments/4CWC/2022/en.md [hunk 4]
```diff
@@ -113,19 +113,19 @@ The 4 Digit Catch World Cup 2022 is run by various community members.
...
-  1. [DJ TOTTO feat. *spiLa* - illumina (Imai Lisa) \[Platter\]](https://osu.ppy.sh/beatmapsets/880513#fruits/2594189)
+  1. [DJ TOTTO feat. \*spiLa\* - illumina (Imai Lisa) \[Platter\]](https://osu.ppy.sh/beatmapsets/880513#fruits/2594189)
   2. [Can't Stop Won't Stop (feat. Fresh Big Mouf) - Mighty & High (thiev) \[Extra\]](https://osu.ppy.sh/beatmapsets/770361#fruits/1952154)
-- Hard Rock
+- HardRock
   1. [Yuzuki - Dear You (DJ Genericname DnB Remix) (R-34) \[R-34 & M0RU's Salad (4CWC Edit.)\]](https://osu.ppy.sh/beatmapsets/1819634#fruits/3733788)
   2. [Yuna (CV: Kanda Sayaka) - Break Beat Bark! (Cut Ver.) (-Crimson-) \[Hard\]](https://osu.ppy.sh/beatmapsets/1285419#fruits/2670654)
-- Double Time
+- DoubleTime
   1. [BLANKFIELD - Retrospective City (Nelly) \[Collab Platter\]](https://osu.ppy.sh/beatmapsets/1203586#fruits/2531997)
```
Comment (ZeroPyrozen) [inline wiki/Tournaments/4CWC/2022/en.md:122]:
```suggestion
  1. [BLANKFIELD - Retrospective City (Nelly) \[Collab Salad\]](https://osu.ppy.sh/beatmapsets/1203586#fruits/2531997)
```

**Review by ZeroPyrozen — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**